src/prelude.rs

```diff
@@ -4,8 +4,8 @@
 
 pub use hibitset::BitSet;
 pub use join::{Join, ParJoin};
-pub use shred::{Dispatcher, DispatcherBuilder, Read, ReadExpect, RunNow, System, SystemData,
-                Write, WriteExpect};
+pub use shred::{Dispatcher, DispatcherBuilder, Read, ReadExpect, Resources, RunNow, System,
+                SystemData, Write, WriteExpect};
```

Comment:
I wonder if we should simply pub crate shred.

Reply:
Honestly given how integral shred is to specs it might not be a bad idea.